EssentialFeed/EssentialFeediOSTests/ImageCommentsSnapshotTests.swift

```diff
@@ -0,0 +1,92 @@
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark)), named: "EMPTY_IMAGE_COMMENTS_DARK")
+	}
+	
+	func test_loadingState() {
+		let sut = makeSUT()
+
+		sut.refreshControl?.beginRefreshing()
+
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light)), named: "IMAGE_COMMENTS_LOADING_LIGHT")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark)), named: "IMAGE_COMMENTS_LOADING_DARK")
+	}
+	
+	func test_commentsWithContent() {
+		let sut = makeSUT()
+		
+		sut.display(commentsWithContent())
+		
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light)), named: "IMAGE_COMMENTS_WITH_CONTENT_LIGHT")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark)), named: "IMAGE_COMMENTS_WITH_CONTENT_DARK")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark, contentSize: .extraExtraLarge)), named: "IMAGE_COMMENTS_WITH_CONTENT_DARK_EXTRA_EXTRA_LARGE")
+	}
+	
+	func test_commentsWithErrorMessage() {
+		let sut = makeSUT()
+		
+		sut.display(ImageCommentsErrorViewModel(message: "This is a\nmulti-line\nerror message"))
+
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light)), named: "IMAGE_COMMENTS_WITH_ERROR_MESSAGE_LIGHT")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark)), named: "IMAGE_COMMENTS_WITH_ERROR_MESSAGE_DARK")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark, contentSize: .extraExtraLarge)), named: "IMAGE_COMMENTS_WITH_ERROR_MESSAGE_DARK_EXTRA_EXTRA_LARGE")
+	}
+	
+	// MARK: - Helpers
+	
+	private func makeSUT() -> ImageCommentsViewController {
+		let bundle = Bundle(for: FeedViewController.self)
+		let storyboard = UIStoryboard(name: "ImageComments", bundle: bundle)
+		let controller = storyboard.instantiateInitialViewController() as! ImageCommentsViewController
+		controller.loadViewIfNeeded()
+		controller.tableView.showsVerticalScrollIndicator = false
+		controller.tableView.showsHorizontalScrollIndicator = false
+		return controller
+	}
+	
+	private func emptyList() -> [ImageCommentCellController] {
+		[]
+	}
+	
+	private func commentsWithContent() -> [ImageCommentCellController] {
+		[
+			ImageCommentViewModel(
+				id: UUID().uuidString,
+				authorUsername: "Ἀριστοτέλης",
+				createdAt: "2020-10-09T11:24:59+0000",
+				message: "But we think that knowledge and the ability to understand belong to knowledge rather than experience, and we believe that people of knowledge are smarter than empiricists, because wisdom depends in all cases on knowledge. And it is because they know the cause and they don't; empiricists know the effect but do not know the cause, and theorists know both the effect and the cause."),
+			ImageCommentViewModel(
+				id: UUID().uuidString,
+				authorUsername: "Ἡράκλειτος ὁ Ἐφέσιος",
+				createdAt: "2020-10-01T04:23:53+0000",
+				message: "μάχεσθαι χρὴ τὸν δῆμον ὑπὲρ τοῦ νόμου ὅκωσπερ τείχεος"),
+			ImageCommentViewModel(
+				id: UUID().uuidString,
+				authorUsername: "This is a username that could exist",
+				createdAt: "2020-09-26T11:22:59+0000",
```

Comment:
Use relative strings in the test data to mimic the real production code.

```suggestion
				createdAt: "3 months ago",
```